packages/flutter_tools/lib/src/isolated/resident_web_runner.dart

```diff
@@ -851,6 +866,10 @@ class _ResidentWebRunner extends ResidentWebRunner {
   @override
   Future<bool> toggleCanvaskit() async {
```

Comment:
This method and all of its implementations can be deleted as it is no longer used